godtools/GeneralExtensions/String.swift

```diff
@@ -15,6 +15,17 @@ extension String {
         return NSLocalizedString(self, tableName: nil, bundle: Bundle.main, value: "", comment: "")
     }
     
+    func removeBreaklines() -> String {
+        let regex = try! NSRegularExpression(pattern: "\n", options: NSRegularExpression.Options.caseInsensitive)
```

Comment:
I'm guessing we don't need the class/struct names `NSRegularExpression.Options` here? I'm not sure though.